### PR TITLE
Adds section for Models with information on ordering attributes.

### PR DESCRIPTION
**Pull request by michaeldupuisjr**

### CODE REVIEW

File: engineering/ember.md [hunk 1]
```diff
@@ -151,6 +152,71 @@ app
...
 ```
 
+## Models
+
+### Organization
+
+Models should be organized as follows:
+
+1. Attributes
+2. Associations
```
Comment (danmcclain) [inline engineering/ember.md]: I lean toward Associations first, but not tied to it. Also, associations is alphabetically before attributes!

**Comment by duggiefresh** [comment]
:+1:

File: engineering/ember.md [hunk 2]
```diff
@@ -151,6 +152,71 @@ app
...
+
+const { computed } = Ember;
+
+export default Model.extend({
+  // Attributes
+  firstName: attr('string'),
+  lastName: attr('string'),
+
+  // Associations
+  children: hasMany('children'),
```
Comment (brzpegasus) [inline engineering/ember.md]: Technically, the model name should be `child` ;):

```js
children: hasMany('child'),
```

Comment (duggiefresh) [inline engineering/ember.md]: Haha. Yeah, c'mon, Mike!

Comment (michaeldupuisjr) [inline engineering/ember.md]: This is not my day for PRs.

File: engineering/ember.md (hunk 1 above)
Comment (michaeldupuisjr) [inline engineering/ember.md]: I will update this to reflect that we don't care about the order in which the groupings appear.

File: engineering/ember.md [hunk 3]
```diff
@@ -151,6 +152,71 @@ app
     serializer.js
 ```
 
+## Models
+
+### Organization
+
+Models should be grouped as follows:
+
+* Attributes
```
Comment (danmcclain) [inline engineering/ember.md]: I actually like some order, I was saying I could be swayed either way. Just want consistency :smile:

Comment (brzpegasus) [inline engineering/ember.md]: I don't have a strong opinion as to what I'd like to see first. That said, attributes, associations, then CPs makes sense.

Comment (michaeldupuisjr) [inline engineering/ember.md]: The way I look at it – I wouldn't want a PR to be held up to re-order the associations or attributes; therefore, I don't think the ordering should be in the styleguides.

Comment (brzpegasus) [inline engineering/ember.md]: True, but as a team and within the same project, it's nice to be consistent.

Comment (duggiefresh) [inline engineering/ember.md]: I like the ordering. Makes parsing out the component easier. 

Kind of like how the `actions` hash should be at the bottom.

Comment (danmcclain) [inline engineering/ember.md]: @duggiefresh This is specifically for models, so there wouldn't be actions. I completely agree that actions object should be at the bottom of the route/controller

Comment (duggiefresh) [inline engineering/ember.md]: Confirm. I should of been moar explicit about the `actions` hash. :tulip:

File: engineering/ember.md [hunk 4]
```diff
@@ -151,6 +152,71 @@ app
...
+// Good
+
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const {
+  Model,
+  attr,
+  hasMany
+} = DS;
```
Comment (brzpegasus) [inline engineering/ember.md]: Update this to use imports.

File: engineering/ember.md [hunk 5]
```diff
@@ -151,6 +152,71 @@ app
...
+// Bad
+
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const {
+  Model,
+  attr,
+  hasMany
+} = DS;
```
Comment (brzpegasus) [inline engineering/ember.md]: Use imports.

**Comment by michaeldupuisjr** [comment]
@brzpegasus Updated to use imports. It also looks like the discussion above regarding ordering does not require changes, so this should be good to merge.

**Pull request merged**